public/styles/styles.css

```diff
@@ -1,10 +1,81 @@
+
+
 body {
   color: #333;
   font-family: Helvetica, Arial, sans-serif;
-  background-color: skyblue; /* Sanity Check! */
 }
 
 h1 {
   margin-top: 100px;
   text-align: center;
 }
+
+.nav-styles {
+  padding: 10px 0;
+  opacity: 0.5;
+}
+
+.nav-list-styles {
+  font-size: 35px;
+  margin: 0 50px;
+}
+
+img {
+  height: 100px;
+  width: 100px;
+}
+
+.movie-image {
+  height: 350px;
+  width: 100%;
+  margin: 50px 25px 15px 25px;
+}
+
+.list-text {
+  font-family: Arial, Helvetica, sans-serif;;
+  text-align: left;
+  font-size: 25px;
+  margin: 0 0 0 25px;
+}
+
+.edit-movies {
+  width: 30%;
+  margin: 0 0 0 25px;
```

Comment:
The fact that you have these same styles on a bunch of classes makes me think you could have used one more class (like .movie-card or something) that was applied to .edit-movies and to .save-movies and to .delete-movies...